### PR TITLE
Add single signon blog post

### PR DESCRIPTION
**Pull request by mooreds**
Mostly pulled from the single sign on guide, but a bit cheekier.

### CODE REVIEW

File: site/_posts/2021-02-04-single-sign-on.md [hunk 1]
```diff
@@ -0,0 +1,655 @@
...
+
+You should see a message like this:
+
+```shell
+v14.4.0
+```
+
+If you have that, you're ready to get started.
+
+## Application domain names
```
Comment (dpolstra) [inline site/_posts/2021-02-04-single-sign-on.md:40]: Application domain names for SSO

File: site/_posts/2021-02-04-single-sign-on.md [hunk 2]
```diff
@@ -0,0 +1,655 @@
...
+
+* The `indexRouter` variable is configured by the `routes/index.js` file. 
+* All `pug` files should live in the `views` directory. 
+* The `pug` view engine will be used for this application.
+* This application has a session lifetime of 60 seconds (the `maxAge` setting expects milliseconds). 
+* `indexRouter` handles the root path, so all application requests fielded by this server will be handled by that code.
+
+Why is the session lifetime so short? Mostly so you can experiment with SSO without waiting for a long time. In a production application, you'd want to tweak this based on your security expectations as well as load expectations. Note that even if a user's session expires in the Pied Piper application, they won't be forced to log in again unless their SSO session has also expired. The routes and views code will be built out in the next sections.
+
+### Creating the routes in index.js
```
Comment (dpolstra) [inline site/_posts/2021-02-04-single-sign-on.md:324]: Creating the SSO login routes in index.js

File: site/_posts/2021-02-04-single-sign-on.md [hunk 3]
```diff
@@ -0,0 +1,655 @@
...
+//...
+/* Login page if we aren't logged in */
+router.get('/login', function (req, res, next) {
+  res.render('login', {title: title +' Login', clientId: clientId, loginUrl: loginUrl});
+});
+//...
+```
+
+This login page is available to anonymous users. For this tutorial, only a link to log in is shown, but, as mentioned above, this page could have text and images illustrating the benefits of this application.
+
```
Comment (dpolstra) [inline site/_posts/2021-02-04-single-sign-on.md:465]: ###Setting up the SSO logout routes

File: site/_posts/2021-02-04-single-sign-on.md [hunk 4]
```diff
@@ -0,0 +1,655 @@
...
+Log in as the user which you registered for both applications. 
+
+You'll be greeted with a welcome message. Now, click the Hooli link and you'll end up at the Hooli homepage without having to authenticate.
+
+You can log out by clicking any of the logout links. One caveat, however. If you are using Safari or Chrome on macOS, multi application logout won't work due to browser quirks with non TLS applications, iframes and cookies. Setting up a proxy and configuring everything to run over TLS will fix this. 
+
+However, logging out with FireFox works whether you use TLS or not.
+
+## Next steps
```
Comment (dpolstra) [inline site/_posts/2021-02-04-single-sign-on.md:645]: Next steps for FusionAuth single sign-on

File: site/_posts/2021-02-04-single-sign-on.md [hunk 5]
```diff
@@ -0,0 +1,655 @@
+---
+layout: blog-post
+title: Setting up single sign-on with FusionAuth
```
Comment (dpolstra) [inline site/_posts/2021-02-04-single-sign-on.md:3]: Setting up single sign-on (SSO) with FusionAuth

Not a huge thing, but we'll be able to hit both keywords if you include them in the title and URL

Comment (mooreds) [reply site/_posts/2021-02-04-single-sign-on.md]: Sounds good!

**Pull request merged**